pom.xml

```diff
@@ -103,6 +103,19 @@
       <groupId>io.jenkins.plugins.mina-sshd-api</groupId>
       <artifactId>mina-sshd-api-core</artifactId>
     </dependency>
+    <!-- this could be optional? -->
+    <dependency>
+      <groupId>net.i2p.crypto</groupId>
+      <artifactId>eddsa</artifactId>
+      <version>0.3.0</version>
+      <optional>true</optional>
+    </dependency>
+    <dependency>
+      <groupId>org.bouncycastle</groupId>
+      <artifactId>bcprov-jdk18on</artifactId>
+      <version>1.77</version>
+      <optional>true</optional>
+    </dependency>
```

Comment:
```suggestion
```

https://github.com/jenkinsci/mina-sshd-api-plugin/pull/98  these are a dependency of mina not the git plugin.

Reply:
done https://github.com/jenkinsci/git-client-plugin/pull/1127/commits/1b368b0e5295076894b6719c582e6302275032f4